qiskit_optimization/deprecation.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2021.
+# (C) Copyright IBM 2021, 2022.
```

Comment:
I modified this flle in the PR but I reverted the change after all. The copyright.py still updates the year. Should I follow the tool, @manoelmarques?

Reply:
If the file is being commited, then you should follow the tool unless you reverted the commit and removed the file from that commit and committed again.

Reply:
Thank you for your information. I could not revert the commit because it includes changes of other files too. So, I follow the tool to update year.